File: PAT (Basic-Level)/C/1067.c
#include <stdio.h>
#include <string.h>

int main(int argc, char const *argv[]){
    char password[21];
    int limitTimes;
    int enterTimes = 0;
    scanf("%s %d", password, &limitTimes);
    getchar(); // clear the '\n', use this because clientEnter could be nothing
    char clientEnter[100];
    while(1){
        //scanf("%s", clientEnter); //the password doesn't contain space but the clientEnter could contain, could not use scanf but gets()
        gets(clientEnter);
        if(strcmp(clientEnter, "#") == 0) //could not use if(clientEnter[0] == '#')
            break;
        enterTimes++;
        if(strcmp(clientEnter, password) != 0 && enterTimes <= limitTimes){
            printf("Wrong password: %s\n", clientEnter);
            if(enterTimes == limitTimes){
                printf("Account locked");
                break;
            }
        }
        else if(strcmp(clientEnter, password) == 0 && enterTimes <= limitTimes){
            printf("Welcome in");
            break;
        }
    }
    return 0;
}

File: PAT (Basic-Level)/Python/1067.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

password, limitTimes = input().split()
limitTimes = int(limitTimes)
enterTimes = 0

while True:
    data = input()
    if data == '#':
        break
    enterTimes += 1
    if data != password and enterTimes <= limitTimes:
        print("Wrong password: %s" % data)
        if enterTimes == limitTimes:
            print("Account locked", end = '')
            break
    elif data == password and enterTimes <= limitTimes:
        print("Welcome in", end = '')
        break
